frontend/src/container/QueryTable/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { RowData } from 'lib/query/createTableColumnsFromQuery';
+
+export function createDownloadableData(
+	inputData: RowData[],
+): Record<string, string>[] {
+	return inputData.map((row) => ({
```

Comment:
Add safety checks wherever required. We should handle safety checks everywhere by default when not strictly typed.

Reply:
Added the safety check.